ui/src/main/java/edu/wpi/grip/ui/pipeline/AddSourceView.java

```diff
@@ -48,6 +49,7 @@
     private final MultiImageFileSource.Factory multiImageSourceFactory;
     private final ImageFileSource.Factory imageSourceFactory;
     private final CameraSource.Factory cameraSourceFactory;
+    private final HttpSource.Factory httpSourceFactory;
```

Comment:
This is wrong; `httpSourceFactory` is used. Looks like codacy doesn't play nice with lambdas.